snapcraft.yaml

```diff
@@ -93,9 +135,9 @@ parts:
       for snap in "core24" "gnome-46-2404"; do
         cd "/snap/$snap/current/usr/lib"
         for filename in [ *.so* ]; do
-          rm -f "$CRAFT_PRIME/usr/lib/$CRAFT_ARCH_TRIPLET/$filename"
+          rm -f "$CRAFT_PRIME/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/$filename"
         done
-        cd "/snap/$snap/current/usr/lib/$CRAFT_ARCH_TRIPLET"
+        cd "/snap/$snap/current/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR"
```

Comment:
Similarly, those changes are for core24 deprecation right? They make sense but aren't part of including libsoup so should be split in another commit

Reply:
Removed. Will send another PR.

Reply:
Also, used libsoup and libgoa packages, as requested.